tests/variant_calling_controlfreec.nf.test

```diff
@@ -0,0 +1,187 @@
+    //     }
+    // }
+
+    // test("Run with profile test | --tools controlfreec | tumoronly") {
+
+    //     when {
+    //         params {
+    //             modules_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
+    //             chr_dir = params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/chromosomes.tar.gz'
+    //             dbsnp = params.modules_testdata_base_path + '/genomics/homo_sapiens/genome/chr21/germlineresources/dbsnp_138.hg38.vcf.gz'
+    //             dbsnp_tbi = params.modules_testdata_base_path + '/genomics/homo_sapiens/genome/chr21/germlineresources/dbsnp_138.hg38.vcf.gz.tbi'
+    //             fasta = params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta'
+    //             fasta_fai = params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai'
+    //             input = "${projectDir}/tests/csv/3.0/recalibrated_tumoronly.csv"
+    //             intervals = params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/multi_intervals.bed'
+    //             outdir = "$outputDir"
+    //             step = "variant_calling"
+    //             tools = 'controlfreec'
+    //             wes = true
+    //         }
+    //     }
+
+    //     then {
+    //         // stable_name: All files + folders in ${params.outdir}/ with a stable name
+    //         def stable_name = getAllFilesFromDir(params.outdir, relative: true, includeDir: true, ignore: ['pipeline_info/*.{html,json,txt}'])
+    //         // stable_path: All files in ${params.outdir}/ with stable content
+    //         def stable_path = getAllFilesFromDir(params.outdir, ignoreFile: 'tests/.nftignore')
+    //         // cram_files: All cram files
+    //         def cram_files  = getAllFilesFromDir(params.outdir, include: ['**/*.cram'])
+    //         def fasta       = params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta'
+    //         assertAll(
+    //             { assert workflow.success},
+    //             { assert snapshot(
+    //                 // Number of successful tasks
+    //                 workflow.trace.succeeded().size(),
+    //                 // pipeline versions.yml file for multiqc from which Nextflow version is removed because we tests pipelines on multiple Nextflow versions
+    //                 removeNextflowVersion("$outputDir/pipeline_info/nf_core_sarek_software_mqc_versions.yml"),
+    //                 // All stable path name, with a relative path
+    //                 stable_name,
+    //                 // All files with stable contents
+    //                 stable_path,
+    //                 // All cram files
+    //                 cram_files.collect{ file -> [ file.getName(), cram(file.toString(), fasta).getReadsMD5() ] }
+    //             ).match() }
+    //         )
+    //     }
+    // }
```

Comment:
why is all of this commented?

Reply:
I can remove them if you prefer, I let them for now, just for a reminder to sort this out later at some point

Reply:
why are we not doing the stub test then?

Reply:
`Cannot invoke method clone() on null object`

Reply:
hm that seems like we are calling something strange somewhere?